### PR TITLE
Validation schema

### PR DESCRIPTION
**Pull request by afaguilarr**
In this branch we developed a test which assures that a response has the correct JSON schema

### CODE REVIEW

File: test/Github.Api.Contract.test.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+const chai = require('chai');
+const { listPublicEventsSchema } = require('./schema/ListPublicEvents.schema');
+
+const { expect } = chai;
+chai.use(require('chai-json-schema'));
+
+const urlBase = 'https://api.github.com';
+
+describe('Given event Github API resources', () => {
+  describe('When wanna verify the List public events', () => {
```
Comment (sorregom) [inline test/Github.Api.Contract.test.js]: when who? you should use `I want to...`

File: test/Github.Api.Contract.test.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    let listPublicEventsQuery;
+
+    before(() => {
+      listPublicEventsQuery = agent
+        .get(`${urlBase}/events`)
+        .auth('token', process.env.ACCESS_TOKEN);
+    });
+
+    it('then the body should have a schema', () =>
```
Comment (sorregom) [inline test/Github.Api.Contract.test.js]: which schema?

**Review by sorregom — CHANGES_REQUESTED** [review]

File: test/Github.Api.Contract.test.js (hunk 1 above)
Comment (afaguilarr) [reply test/Github.Api.Contract.test.js]: @sorregom D: así estaba, eso se copiaba directamente del taller, ya lo corrijo

**Review by sorregom — APPROVED** [review]
LGTM

**Pull request merged**